### PR TITLE
Disable Artifact Binding

### PR DESCRIPTION
**Pull request by VivekVinushanth**
### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 1]
```diff
@@ -890,36 +906,40 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
-                                    disabled={ !isArtifactBindingEnabled }
-                                    data-testid={ `${ testId }-artifact-binding-signature-validation-checkbox` }
-                                />
-                                <Hint disabled={ !isArtifactBindingEnabled }>
-                                   { t("console:develop.features.applications.forms.inboundSAML.sections" +
-                                           ".ssoProfile.fields.artifactBinding.hint") }
-                                </Hint>
-                            </Grid.Column>
-                        </Grid.Row>
+                        {isArtifactBindingAllowed
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:909]: Lets wrap the multiline JSX with `()`
```suggestion
                        {
                            isArtifactBindingAllowed
                                ? (
                                    JSX
                                )
                                : null
```

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 2]
```diff
@@ -276,6 +277,33 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
         }, [initialValues]
     );
 
+    useEffect(() => {
+        setIsArtifactBindingAllowed(applicationConfig.inboundSAMLForm.artifactBindingAllowed);
+        }, []
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:282]:
```suggestion
    }, []);
```

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx (hunk 1 above)
Comment (VivekVinushanth) [reply apps/console/src/features/applications/components/forms/inbound-saml-form.tsx]: Addressed with aaa965ccd1f35f4c6c2c998bf202ab99a9f7d5ef

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx (hunk 2 above)
Comment (VivekVinushanth) [reply apps/console/src/features/applications/components/forms/inbound-saml-form.tsx]: Addressed with aaa965ccd1f35f4c6c2c998bf202ab99a9f7d5ef

**Pull request merged**